package.json

```diff
@@ -63,6 +61,7 @@
     "request": "^2.81.0"
   },
   "dependencies": {
+    "argon2": "^0.15.0",
```

Comment:
Because Argon2 won the [Password Hashing Competition](https://password-hashing.net) and they say, "We recommend that use you use Argon2 rather than legacy algorithms."
Scrypt: :woman_shrugging: 
Blowfish: [no longer recommended](https://en.wikipedia.org/wiki/Blowfish_(cipher)#Weakness_and_successors)
Many SHA-256: how do we know we've done enough? Why not just use an algorithm that is known to generate good hashes.